src/components/App/App.component.jsx

```diff
@@ -7,36 +8,28 @@ import LoginPage from '../../pages/Login';
 import NotFound from '../../pages/NotFound';
 import SecretPage from '../../pages/Secret';
 import Private from '../Private';
-import Fortune from '../Fortune';
+// import Fortune from '../Fortune';
 import Layout from '../Layout';
-import { random } from '../../utils/fns';
-
-function App() {
-  useLayoutEffect(() => {
-    const { body } = document;
 
-    function rotateBackground() {
-      const xPercent = random(100);
-      const yPercent = random(100);
-      body.style.setProperty('--bg-position', `${xPercent}% ${yPercent}%`);
-    }
+//Components
+import { Header } from '../../components/Header/Header.component';
+import { VideoDetail } from '../../components/VideoDetail/VideoDetail.component';
 
-    const intervalId = setInterval(rotateBackground, 3000);
-    body.addEventListener('click', rotateBackground);
+function App() {
 
-    return () => {
-      clearInterval(intervalId);
-      body.removeEventListener('click', rotateBackground);
-    };
-  }, []);
+  const [searchValue, setSearchValue] = useState('doggos');
```

Comment:
This looks good, in the next challenge you'll have an easy time setting the context